examples/config/FollowupParameters_SWIFT.ini

```diff
@@ -0,0 +1,34 @@
+[observatory]
+base = space
+name = SWIFT
```

Comment:
It is not SWIFT, but Swift

Reply:
No it will not work as the database needs the name of the observatory in capital letters